packages/client-sdk-web/package.json

```diff
@@ -48,7 +48,7 @@
     "xhr2": "^0.2.1"
   },
   "dependencies": {
-    "@gomomento/common": "file:../common",
+    "@gomomento/core": "file:../common",
```

Comment:
lol I feel like we keep changing this. `core` makes more sense to me if we are publishing it to npm